In [1]:
import math
import scipy.stats as stats
import numpy as np

In [2]:
sample_data = [6414,1550,2109,9350,21828,4300,5944,5722,2825,2044,5481,5200,5853,2750,10012,6357,27000,9414,76813200,17500,9200,7380,18314,6557,13713,17768,7493,2771,2861,
1263,7285,28165,5080,11622]

In [4]:
sample_size = len(sample_data)
print('The sample size of the data is:',sample_size)

The sample size of the data is: 34


In [8]:
sample_mean = round(np.mean(sample_data),2)
print('The sample mean of the data is:',sample_mean)

The sample mean of the data is: 2267774.26


In [9]:
sample_std = np.std(sample_data, ddof=1)
sample_std = round(sample_std,2)
print("Our std for this data is:",sample_std)

Our std for this data is: 13171844.01


In [10]:
std_err = sample_std / math.sqrt(sample_size)
std_err = round(std_err,2)
print("The std_error for the data is:", std_err)

The std_error for the data is: 2258952.61


In [11]:
t_value = stats.t.ppf(0.975,33)
print("t critical value = ", round(t_value,2))

t critical value =  2.03


In [12]:
margin_of_error = round(t_value * std_err,2) 
print("The margin of error for this data is:", margin_of_error)

The margin of error for this data is: 4595873.64


In [14]:
lower = sample_mean - margin_of_error
upper = sample_mean + margin_of_error

print("The lower limit for this data is:", lower)
print("The upper limit for this data is:", round(upper,2))

The lower limit for this data is: -2328099.38
The upper limit for this data is: 6863647.9


# You will see above that the 95% Confidence level for this list of data is -2328099.38 at the lower end and  6863647.9 as the higher end.